src/main/java/co/rsk/bitcoinj/script/RedeemScriptValidator.java

```diff
@@ -219,8 +229,35 @@ protected static List<ScriptChunk> removeOpCheckMultisig(Script redeemScript) {
         return redeemScript.getChunks().subList(0, redeemScript.getChunks().size() - 1);
     }
 
-    protected static boolean isOpN(ScriptChunk chunk) {
-        return chunk.isOpCode() &&
-            chunk.opcode >= ScriptOpCodes.OP_1 && chunk.opcode <= ScriptOpCodes.OP_16;
+    public static int decodeN(ScriptChunk chunk) {
```

Comment:
Missing unit tests

Reply:
Should this go here? This class is a RedeemScriptValidator and it's exposing a method to decode a chunk to a number, not sure I follow why

Reply:
Moved to ScriptChunk class